VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -163,8 +163,7 @@ private void setCurrentUnit(CourseComponent component){
         resultData.putExtra(Router.EXTRA_COURSE_COMPONENT_ID, courseComponentId);
         setResult(RESULT_OK, resultData);
 
-        boolean isPortrait = getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT;
-        environment.getSegment().trackCourseComponentViewed(selectedUnit.getId(), courseData.getCourse().getId(), isPortrait);
+        environment.getSegment().trackCourseComponentViewed(selectedUnit.getId(), courseData.getCourse().getId());
```

Comment:
Since this exact call is being used multiple times in this class, we might create a local utility method for it.